recipes/clhep/all/conanfile.py

```diff
@@ -52,6 +44,7 @@ def validate(self):
 
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        apply_conandata_patches(self)
```

Comment:
will do, but, can I ask - given that this applies to all feasible build configurations, why?

Reply:
Immutability of source code. Some patches may be conditional (depending on os, compiler etc), so it cannot be done in source(). conancenter prefers to keep all the modification of source code in the same place.

Reply:
I understand that, but none of _these_ patches are conditional (notwithstanding 2 of them are for windows compatibility, they get applied to all configurations anyway).  For myself in this case then I'd prefer it in source() because precisely because it conveys that meaning.  You have the original source in your cache as well anyway, at least in the core.sources:download_cache ;-) .

But I'm not arguing of course if that's the rule that's the rule.